srcpkgs/mpg123-extras/template

```diff
@@ -1,23 +1,21 @@
 # Template file for 'mpg123-extras'
 # Sync with mpg123
 pkgname=mpg123-extras
-version=1.31.2
+version=1.31.3
 revision=1
 build_style=gnu-configure
 # --with-cpu not necessary, upstream detects features correctly
-configure_args="--with-optimization=0 --with-default-audio=alsa"
```

Comment:
Pretty sure we set `--with-optimization=0` so that it respects the default of `xbps-src`.

Reply:
Thanks for the heads-up, I put it back with an explanatory comment.